plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/util/TestingHadoop.java

```diff
@@ -1,271 +0,0 @@
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.deltalake.util;
-
-import com.github.dockerjava.api.command.CreateContainerCmd;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
-import io.airlift.log.Logger;
-import io.trino.testing.containers.wait.strategy.SelectedPortWaitStrategy;
-import net.jodah.failsafe.Failsafe;
-import net.jodah.failsafe.RetryPolicy;
-import org.apache.hadoop.net.NetUtils;
-import org.testcontainers.containers.Container;
-import org.testcontainers.containers.Network;
-import org.testcontainers.containers.startupcheck.IsRunningStartupCheckStrategy;
-import org.testcontainers.utility.MountableFile;
-
-import java.io.Closeable;
-import java.io.IOException;
-import java.time.Duration;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.function.Consumer;
-
-import static com.google.common.base.Preconditions.checkArgument;
-import static java.lang.String.format;
-import static java.time.temporal.ChronoUnit.MINUTES;
-import static java.time.temporal.ChronoUnit.SECONDS;
-import static java.util.Objects.requireNonNull;
-
-public class TestingHadoop
-        implements Closeable
-{
-    private static final Logger log = Logger.get(TestingHadoop.class);
-    public static final String HADOOP_MASTER = "hadoop-master";
-
-    public static final int METASTORE_PORT = 9083;
-    public static final int PROXY_PORT = 1180;
-    public static final int HIVE_SERVER_PORT = 10000;
-
-    public static final Set<Integer> HIVE_PORTS = ImmutableSet.of(
-            METASTORE_PORT,
-            PROXY_PORT,
-            HIVE_SERVER_PORT);
-
-    private final DockerContainer container;
-    private final Closer closer = Closer.create();
-
-    private TestingHadoop(
-            String image,
-            Set<Integer> ports,
-            Map<String, String> resourcesToMount,
-            Map<String, String> filesToMount,
-            Map<String, String> extraHosts,
-            Optional<Network> network,
-            int retryLimit)
-    {
-        RetryPolicy<Object> retryPolicy = new RetryPolicy<>()
-                .withMaxRetries(retryLimit)
-                .onRetry(event -> log.warn(
-                        "TestingHadoop initialization failed (attempt %s), will retry. Exception: %s",
-                        event.getAttemptCount(),
-                        event.getLastFailure().getMessage()));
-
-        container = Failsafe.with(retryPolicy).get(() -> startHadoopContainer(image, ports, resourcesToMount, filesToMount, extraHosts, network));
-        closer.register(container::stop);
-
-        // HACK: even though Hadoop is accessed by proxy, Hadoop still tries to resolve hadoop-master
```

Comment:
@losipiuk I just want to make sure that this logic is not needed with the transition towards `HiveHadoop` where this setting is not made anymore.